### ISSUE TITLE
Ensure tables have labels

### ISSUE THREAD

**Issue by nwalters512** [PrairieLearn/PrairieLearn#10489]
Tables should have descriptive labels so that they can be easily understood by screen readers.

For labels, we have a few options:

- We can use a `<caption>...</caption>` within the table, possibly with a `sr-only` class so that it's not visible on screen.
- We can use `aria-label` or `aria-labelledby` on the table itself.

We should test both options with screen readers. I'd lean towards using `aria-label`.

We can also consider adding `aria-describedby` to tables with more complex data to more fully describe them. I'm not sure if this is worth it.
